ideas/92-disc-v5-research.md

```diff
@@ -44,8 +44,8 @@ Description:
 1. If so, there is a list of issues that we should work on to further improve DiscV5 usage in the Status app and in the cluster (this may be realized as next iterations or a new swarm).
 
 ## Success Metrics
-1. It's possible to start statusd/Status App with bootnodes and DiscV5 enabled,
-1. It's possible to discover LES/2, Whisper and MailServer nodes via DiscV5 (nodes with the same protocol but different capabilities),
+1. Connecting to the network for statusd/Status App via DiscV5 is successful in at least 99% of tries,
+1. It's possible to discover minimal number of LES/2, Whisper and MailServer nodes via DiscV5 within 15 seconds without cache and in 5 seconds with cache,
```

Comment:
Oh, good hint, will become interesting when LES/3 nodes in future offer payed services with different capabilities. Has to be supported by a *DiscVx* later too.